In [2]:
import pandas as pd

In [3]:
#Cargado de archivos
#diabetes_mellitus  =  pd.read_excel('archivos/diabetes_mellitus.xlsx')
#matrimonios = pd.read_excel('archivos/matrimonios.xlsx')
#porcentaje_nacimientos = pd.read_excel('archivos/porcentaje_nacimientos.xlsx')
#tasaMortalidadInfantil = pd.read_excel('archivos/tasaMortalidadInfantil.xlsx')
#nacimientosResidenciaMadre = pd.read_excel('archivos/NacimientosResidenciaMadre.xlsx')
#mortalidadExterna = pd.read_excel('archivos/MortalidadExterna.xlsx')
#porcentajePesoNacer = pd.read_excel('archivos/PorcentajePesoNacer.xlsx')
#divorcios = pd.read_excel('archivos/Divorcios.xlsx')

mortalidadVIH_SIDA = pd.read_excel('archivos/MortalidadVIH_SIDA.xlsx')
defunciones = pd.read_excel('archivos/defunciones2.xlsx')

defunciones


Depreg  Mupreg  Mesreg  Añoreg  Depocu  Mupocu  Areag  Sexo  Diaocu  \
0           5     505       1       9       5     505      9     1       2   
1           1     101       9       9       1     101      9     1      27   
2          22    2206       9       9      22    2206      9     2      23   
3           2     201      12       9       2     201      9     1       5   
4           1     101       5       9       1     101      9     2       7   
...       ...     ...     ...     ...     ...     ...    ...   ...     ...   
71702      15    1501       8       9      15    1501      1     2      28   
71703       9     901       4       9       9     901      1     2      31   
71704      17    1711      11       9      17    1711      1     1       2   
71705      16    1601       7       9      16    1601      1     2       3   
71706      17    1703       3       9      17    1703      1     2      19   

       Mesocu  ...  Ocudif  Dnadif  Mnadif  Nacdif  Dredif  Mredif  Caudef  \
0           1  ...    9712       1     101       0       1     101    Y214   
1           9  ...    9712       1     101       0       1     101    X919   
2           8  ...    9712       1     101       0       1     101    X959   
3          12  ...    9999       1     101       0       1     101    V899   
4           5  ...    9712       1     101       0       1     101    X709   
...       ...  ...     ...     ...     ...     ...     ...     ...     ...   
71702       8  ...    9999      99    9999       0      99    9999    P025   
71703       3  ...    9999      99    9999       0      99    9999    P025   
71704      11  ...    9999      99    9999       0      99    9999    P017   
71705       7  ...    9999      99    9999       0      99    9999    J969   
71706       3  ...    9999      99    9999       0      99    9999    J80X   

       Asist  Ocur  Cerdef  
0          1     3       1  
1          4     3       1  
2          4     3       2  
3          4     3       1  
4          4     3       1  
...      ...   ...     ...  
71702      1     1       1  
71703      1     1       1  
71704      1     1       1  
71705      1     1       1  
71706      1     1       1  

[71707 rows x 25 columns]

In [4]:

mortalidadVIH_SIDA.describe()
defunciones.describe()


Depreg        Mupreg        Mesreg        Añoreg        Depocu  \
count  71707.000000  71707.000000  71707.000000  71707.000000  71707.000000   
mean       8.605757    866.551034      6.430321      9.016135      8.605073   
std        6.687519    669.976160      3.435892      0.125996      6.687544   
min        1.000000    101.000000      1.000000      9.000000      1.000000   
25%        1.000000    108.000000      3.000000      9.000000      1.000000   
50%        9.000000    901.000000      6.000000      9.000000      9.000000   
75%       14.000000   1406.000000      9.000000      9.000000     14.000000   
max       22.000000   2217.000000     12.000000     10.000000     22.000000   

             Mupocu         Areag          Sexo        Diaocu        Mesocu  \
count  71707.000000  71707.000000  71707.000000  71707.000000  71707.000000   
mean     866.543294      1.680031      1.423292     15.629520      6.475658   
std      669.891405      1.401926      0.494084      8.806247      3.430338   
min      101.000000      1.000000      1.000000      1.000000      1.000000   
25%      108.000000      1.000000      1.000000      8.000000      4.000000   
50%      901.000000      1.000000      1.000000     16.000000      6.000000   
75%     1406.000000      2.000000      2.000000     23.000000      9.000000   
max     2217.000000      9.000000      2.000000     31.000000     12.000000   

       ...        Ecidif        Ocudif        Dnadif        Mnadif  \
count  ...  71707.000000  71707.000000  71707.000000  71707.000000   
mean   ...      1.664803   9066.538985     16.278271   1682.061654   
std    ...      1.501258   1437.087100     24.822250   2565.880825   
min    ...      1.000000    110.000000      1.000000    101.000000   
25%    ...      1.000000   9211.000000      3.000000    306.000000   
50%    ...      1.000000   9711.000000     10.000000   1013.000000   
75%    ...      2.000000   9712.000000     16.000000   1612.000000   
max    ...      9.000000   9999.000000     99.000000   9999.000000   

             Nacdif        Dredif        Mredif        Asist          Ocur  \
count  71707.000000  71707.000000  71707.000000  71707.00000  71707.000000   
mean       1.096309     11.753134   1184.903287      2.70869      3.102640   
std        9.104364     16.100507   1624.015508      1.50075      1.391381   
min        0.000000      1.000000    101.000000      1.00000      1.000000   
25%        0.000000      3.000000    301.000000      1.00000      1.000000   
50%        0.000000      9.000000    922.000000      4.00000      4.000000   
75%        0.000000     15.000000   1504.000000      4.00000      4.000000   
max       99.000000     99.000000   9999.000000      9.00000      9.000000   

             Cerdef  
count  71707.000000  
mean       2.724574  
std        3.184217  
min        1.000000  
25%        1.000000  
50%        1.000000  
75%        2.000000  
max        9.000000  

[8 rows x 24 columns]

In [5]:
import warnings
warnings.filterwarnings('ignore')


#Mostrar información general del dataframe

defunciones.info()
defunciones.shape



<class 'pandas.core.frame.DataFrame'>
RangeIndex: 71707 entries, 0 to 71706
Data columns (total 25 columns):
 #   Column  Non-Null Count  Dtype 
---  ------  --------------  ----- 
 0   Depreg  71707 non-null  int64 
 1   Mupreg  71707 non-null  int64 
 2   Mesreg  71707 non-null  int64 
 3   Añoreg  71707 non-null  int64 
 4   Depocu  71707 non-null  int64 
 5   Mupocu  71707 non-null  int64 
 6   Areag   71707 non-null  int64 
 7   Sexo    71707 non-null  int64 
 8   Diaocu  71707 non-null  int64 
 9   Mesocu  71707 non-null  int64 
 10  Añoocu  71707 non-null  int64 
 11  Edadif  71707 non-null  int64 
 12  Perdif  71707 non-null  int64 
 13  Getdif  71707 non-null  int64 
 14  Ecidif  71707 non-null  int64 
 15  Ocudif  71707 non-null  int64 
 16  Dnadif  71707 non-null  int64 
 17  Mnadif  71707 non-null  int64 
 18  Nacdif  71707 non-null  int64 
 19  Dredif  71707 non-null  int64 
 20  Mredif  71707 non-null  int64 
 21  Caudef  71707 non-null  object
 22  Asist   71707 non-null

(71707, 25)

El conjunto defunciones tiene 71707 observaciones.


El tipo de cada varibale para cada columna es:

Depreg: cuantitativa discreta, representa la edad de la madre al momento del parto.

Mupreg: cuantitativa discreta, representa la edad del padre al momento del parto.

Mesreg: cuantitativa discreta, representa el mes del registro del nacimiento.

Añoreg: cuantitativa discreta, representa el año del registro del nacimiento.

Depocu: cuantitativa discreta, representa la edad de la madre en años cumplidos al 31 de diciembre del año del registro.

Mupocu: cuantitativa discreta, representa la edad del padre en años cumplidos al 31 de diciembre del año del registro.

Areag: cualitativa nominal, representa el área geográfica donde se registró el nacimiento (urbana o rural).

Sexo: cualitativa nominal, representa el sexo del recién nacido (masculino o femenino).

Diaocu: cuantitativa discreta, representa el día del registro del nacimiento.

Mesocu: cuantitativa discreta, representa el mes del registro del nacimiento.

edadif: variable cuantitativa que representa la diferencia de edad entre dos personas o eventos.

perdif: variable cualitativa nominal que indica el tipo de persona que experimentó la pérdida. L

getdif: variable cuantitativa que representa la diferencia entre dos fechas o eventos en términos de años.

Ecidif: cualitativa nominal, representa el establecimiento de salud donde ocurrió el nacimiento.

Ocudif: cualitativa nominal, representa la ocupación de la madre al momento del parto.

Dnadif: cualitativa nominal, representa la educación del padre.

Mnadif: cualitativa nominal, representa la educación de la madre.

Nacdif: cualitativa nominal, representa la nacionalidad de la madre.

Dredif: cualitativa nominal, representa la educación del responsable de la declaración del nacimiento.

Mredif: variable cualitativa nominal que indica el tipo de relación entre dos personas o eventos que experimentaron la pérdida. 

Caudef: variable cualitativa nominal que indica la causa de la muerte de una persona. 

Asist: variable cualitativa nominal que indica si se recibió o no asistencia después de la pérdida.

Ocur: variable cualitativa nominal que indica si la pérdida fue reciente o no.

Cerdef: variable cualitativa nominal que indica si se tiene o no certeza sobre la muerte de una persona.


In [ ]:
mortalidadVIH_SIDA.info()
